client/landing/jetpack-cloud/components/threat-item-header/index.tsx

```diff
@@ -8,49 +8,79 @@ import { translate } from 'i18n-calypso';
+const ThreatItemHeader = ( { threat, isStyled = true }: Props ) => {
 	const { filename, extension = { slug: 'unknown', version: 'n/a' } } = threat;
 
 	const basename = filename ? filename.replace( /.*\//, '' ) : '';
 
 	switch ( getThreatType( threat ) ) {
 		case 'core':
-			return translate( 'Infected core file: {{filename/}} ', {
-				components: {
-					filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
-				},
-			} );
+			return isStyled
+				? translate( 'Infected core file: {{filename/}}', {
+						components: {
+							filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
+						},
+				  } )
+				: translate( 'Infected core file: %s', {
+						args: [ basename ],
+				  } );
 
 		case 'file':
-			return translate( 'The file {{filename/}} contains a malicious code pattern.', {
-				components: {
-					filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
-				},
-			} );
+			return isStyled
+				? translate( 'The file {{filename/}} contains a malicious code pattern', {
+						components: {
+							filename: <code className="threat-item-header__alert-filename">{ basename }</code>,
+						},
+				  } )
+				: translate( 'The file %s contains a malicious code pattern', {
+						args: [ basename ],
+				  } );
 
 		case 'plugin':
-			return translate( 'Vulnerable Plugin: {{pluginSlug/}} (version {{version/}})', {
-				components: {
-					pluginSlug: <span className="threat-item-header__alert-slug">{ extension.slug }</span>,
-					version: <span className="threat-item-header__alert-version">{ extension.version }</span>,
-				},
-			} );
+			return isStyled
+				? translate( 'Vulnerable Plugin: {{pluginSlug/}} (version {{version/}})', {
+						components: {
+							pluginSlug: (
+								<span className="threat-item-header__alert-slug">{ extension.slug }</span>
+							),
+							version: (
+								<span className="threat-item-header__alert-version">{ extension.version }</span>
+							),
+						},
+				  } )
+				: translate( 'Vulnerable Plugin: %(pluginSlug)s (version %(version)s)', {
+						args: {
+							pluginSlug: extension.slug,
+							version: extension.version,
+						},
+				  } );
 
 		case 'theme':
-			return translate( 'Vulnerable Theme {{themeSlug/}} (version {{version/}})', {
-				components: {
-					themeSlug: <span className="threat-item-header__alert-slug">{ extension.slug }</span>,
-					version: <span className="threat-item-header__alert-version">{ extension.version }</span>,
-				},
-			} );
+			return isStyled
+				? translate( 'Vulnerable Theme {{themeSlug/}} (version {{version/}})', {
+						components: {
+							themeSlug: <span className="threat-item-header__alert-slug">{ extension.slug }</span>,
+							version: (
+								<span className="threat-item-header__alert-version">{ extension.version }</span>
+							),
+						},
+				  } )
+				: translate( 'Vulnerable Theme %(themeSlug)s (version %(version)s)', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **9** times:
`translate( 'Vulnerable Theme {{themeSlug/}} (version {{version/}})' )` *ES Score: 8*